.github/workflows/validatePR.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: dotnet test Src/CSharpier.Tests/CSharpier.Tests.csproj -c release
-  check_formatting:
-    runs-on: ubuntu-latest
-    name: Check Formatting
-    steps:
-      - uses: actions/checkout@v2
-      - run: |
-          dotnet tool restore
-          dotnet csharpier --check
+#  check_formatting:
```

Comment:
Commented out code in revision control is generally not a good thing :smirk:

Reply:
I think this is the last bug that has been screwing me over because when I format code locally it doesn't match how this action thinks the code should be formatted.